docs/fomega/deep-isorecursive/alternatives.md

```diff
@@ -74,25 +82,15 @@ cons  = /\(a :: *) -> \(x : a) (xs : list a) ->
             iwrap listF a /\(r :: *) -> \(z : r) (f : a -> list a -> r) -> f x xs
 ```
 
-This is basically the same encoding as with the head-spine form approach, except `a` is provided directly rather than via a single-element list. Of course, since we do not have spines now, we can't straightforwardly define the `nat` data type anymore: you have to put some argument into `ifix` and `iwrap`. We can put a dummy argument there, though, for example `all (a :: *). a`. The encoding then look like this:
-
-```
-dummy = all (a :: *). a
-natF  = \(nat :: * -> *) (dum :: *) -> all r. r -> (nat dum -> r) -> r
-nat   = ifix natF dummy
-zero  = iwrap natF dummy /\(r :: *) -> \(z : r) (f : nat -> r) -> z
-succ  = \(n : nat) -> iwrap natF dummy /\(r :: *) -> \(z : r) (f : nat -> r) -> f n
-```
-
-That `dummy` argument is never used and is just passed around for the sole reason that we need to apply `ifix` and `iwrap` to some type argument.
+This is basically the same encoding as with the head-spine form approach, except `a` is provided directly rather than via a single-element list.
 
 However we also need to cover the generic n-ary case. With kind-level products we could write `fix f (a1, a2, ... an)`. We do not have kind-level products, but we can just Church-encode spines:
 
 ```
 ifix f (\(r :: k1 -> k2 -> ... -> kn -> *) -> r a1 a2 ... an)
 ```
 
-which gives us `k ~ (k1 -> k2 -> ... -> kn -> *) -> *`. This is not a "true" Church-encoded spine, because the resulting type is limited to be of kind `*`, but this seems enough in our case ([an illustration](https://gist.github.com/effectfully/e57d2816c475928a380e5a6b897ad17d#file-ifixn-agda)).
+which gives us `k ~ (k1 -> k2 -> ... -> kn -> *) -> *`. This is not a "true" Church-encoded spine, because the resulting type is limited to be of kind `*`, but this seems enough in our case (see [this illustration](https://github.com/input-output-hk/plutus/blob/a51f2e8bbfad1ce22b3769b1bac9a0b054cadbfd/docs/fomega/deep-isorecursive/TreeForest.md)).
```

Comment:
Is it possible to use some relative path links here rather than linking to specific revisions of the repo?

Reply:
Yes, but I fear we can move files around and break links. Though, broken links is probably better than outdated docs. I'll change that.